velox/type/Timestamp.h

```diff
@@ -186,7 +190,7 @@ void parseTo(folly::StringPiece in, ::facebook::velox::Timestamp& out);
 
 template <typename T>
 void toAppend(const ::facebook::velox::Timestamp& value, T* result) {
-  // TODO Implement
+  result->append(value.toString());
```

Comment:
Would you add a small test for the code path to velox/type/tests/TimestampTest.cpp?

Reply:
yes